### PR TITLE
[components/Nav][xs]: Discord Icon in Nav links to discord channel

### PR DESCRIPTION
**Pull request by PhilippeduPreez**
URL of discord channel can be changed in config discord field

### CODE REVIEW

**Comment by PhilippeduPreez** [comment]
@olayway I just want to make sure the URL I added for discord is the one we want?

File: templates/default/components/Nav.jsx [hunk 1]
```diff
@@ -15,6 +15,21 @@ function GitHubIcon(props) {
...
 }
 
+function DiscordIcon(props) {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="16"
+      height="16"
+      fill="currentColor"
+      class="bi bi-discord"
```
Comment (PhilippeduPreez) [inline templates/default/components/Nav.jsx:25]: I'm going to remove the class here

File: site/content/config.js [hunk 2]
```diff
@@ -26,6 +26,7 @@ export const config = {
     { href: "/_all", name: "All" },
   ],
   github: "https://github.com/flowershow/flowershow",
+  discord: "https://discord.com/channels/1029670148824641576/1029670149470556172",
```
Comment (PhilippeduPreez) [inline site/content/config.js:29]: I just used the link I got from the channel but please let me know if this should be different

**Review by olayway — APPROVED** [review]
LGTM :)

**Pull request merged**